uint8            type
bool             is_valid
float64          length
float64          width
ros_leap/Vector3 center
ros_leap/Vector3 direction
ros_leap/Vector3 prev_joint
ros_leap/Vector3 next_joint

#bone_names = ['Metacarpal', 'Proximal', 'Intermediate', 'Distal']